bindings/nodejs/rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+tab_spaces = 2
+edition = "2021"
```

Comment:
This should match the `rustfmt.toml` in the Rust directory and enforce licence headers.